packages/clay-multi-step-nav/src/MultiStepNavWithBasicItems.tsx

```diff
@@ -0,0 +1,51 @@
+/**
+ * © 2019 Liferay, Inc. <https://liferay.com>
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import ClayMultiStepNav from '../src/MultiStepNav';
+import React from 'react';
+
+interface ISteps {
+	subTitle?: string;
+	title?: string;
+}
+
+interface IProps {
+	activeIndex: number;
+	onIndexChange: (val: number) => void;
+	spritemap?: string;
+	steps: Array<ISteps>;
+}
+
+export const ClayMultiStepNavWithBasicItems: React.FunctionComponent<
```

Comment:
Probably we have to add this in the high-levels: When there are a certain amount of steps they should live inside the DropDown, as well as the cases of pagination and breadcrumb... Maybe we can add something to the low levels too...

Reply:
👍